hawkular-inventory-bus/src/test/java/org/hawkular/inventory/bus/BusTest.java

```diff
@@ -228,9 +238,11 @@ public void sendMetricEvent() throws Exception {
 //        broker.start();
```

Comment:
> This used to work - either explain why the test is no longer needed or make it work again, pls.

Not sure I follow. Are you referring to the `//        broker.start();` I didn't touch that.